core-bundle/src/Resources/contao/templates/_new/content_element/html.html.twig

```diff
@@ -0,0 +1,7 @@
+{% if as_editor_view %}
+    {% block overview %}
```

Comment:
What's that block? Is it needed? And if so, it should be renamed, no?

Reply:
Well spotted :smile: I renamed it to `editor_view` in ab781239abc0adc297a273e18d0dbe50fd9c58cc.

IMHO it should exist, so that you can adjust the editor view only. Especially if you extend multiple times.